### ISSUE TITLE
Detailed test messages with XML special characters are incomplete

### ISSUE THREAD

**Issue by lgoltz** [opengeospatial/teamengine#278]
As reported with https://github.com/opengeospatial/ets-wmts10-nsg/issues/28 messages containing special XML characters like `<` or `>` are incomplete. For example: 
`WMTS ServiceMetadata Capabilities docuemnt contains no <Keyword> Elements under the <Keywords>.` becomes 
`WMTS ServiceMetadata Capabilities docuemnt contains no Elements under the .`

**Comment by dstenger** [comment]
@keshav-nangare 
Can you please do further investigations?

**Comment by dstenger** [comment]
If you use ```view-source:``` in Firefox, the bug can be identified:
```
            <tr bgcolor="#FFB2B2">
               <td><a href="wmtsCapabilitiesLayerStyleLegends84.html#wmtsCapabilitiesLayerStyleLegends84"
                     class="testDetailsLink"
                     id="wmtsCapabilitiesLayerStyleLegends84.html#wmtsCapabilitiesLayerStyleLegends84">wmts Capabilities Layer Style Legends </a></td>
               <td>
                  <p> The following asserts failed:
                     	<Layer>: Airports.Airports does not have a minScaleDenominator for <Style>: default for a Legend 
```

The ```Layer```and ```Style``` elements are treated as HTML elements.

In general, there are several other syntax errors in the HTML. These are corrected by Firefox automatically.

**Comment by keshav-nangare** [comment]
Fix is included in #277.

**Comment by lgoltz** [comment]
Pull request #277 solves the problem . Successfully tested with locale docker environment.
